### DISCUSSION TITLE
Security issue on Github pages

### DISCUSSION THREAD

**Discussion by ghost** [community/community#39445] [Repositories]
### Select Topic Area

Bug

### Body

All my repositories on Github Pages have this problem
(https://i.stack.imgur.com/afWSK.jpg)

Here is a github pages example:
https://gabriel-bsm.github.io/formulario-unicodigo/

Everything was normal, I decided to create a repository for a project that would be a perfect clone of the Facebook registration page, after I uploaded this application to my Github Pages on the same day this happened.

I deleted this repository, but the problem persists.

**Comment by airtower-luna** [comment]
Looks like your site ended up on the [Google Safe Browsing blocklist](https://safebrowsing.google.com/), which is perfectly reasonable if you make such a clone, and it'll usually affect your whole `github.io` domain. Check [their documentation](https://developers.google.com/search/docs/monitor-debug/security/social-engineering?hl=en) for how to request a review. Obviously that only makes sense after removing the cloned site.
